zerver/lib/outgoing_webhook.py

```diff
@@ -340,3 +340,10 @@ def do_rest_call(base_url: str,
         fail_with_message(event, response_message)
         notify_bot_owner(event, exception=e)
         return None
+
+    except JsonableError as e:
+        response_message = (e.msg)
+        logging.exception("Outhook trigger failed:", stack_info=True)
+        fail_with_message(event, response_message)
+        notify_bot_owner(event, exception=e)
+        return None
```

Comment:
I think it might be more appropriate to put this in a separate try/except around the `process_success_response`, since we are aiming to scope this for the request to send a message.  This will allow us to provide better error text (as described below).

Reply:
That makes sense. Will do.